portfolio/src/main/webapp/index.html

```diff
@@ -9,31 +9,95 @@
+      
+      <div id = "hero-section">
+          <div id = "hero-text">
+              <h1> 👋 Hi! I'm <span class = "accent">Maxwell</span>. </h1>
+              <p id = "msg"> I'm a rising junior at UC Berkeley studying <span class = "accent">Computer Science and Bioengineering</span>. I love music, cooking, and teaching!</p>
+          </div>
+          <div id = "hero-img">
+            <img id = "profile-pic" alt = "Photo of Maxwell" src = "/images/Maxwell.jpg">
+          </div>
       </div>
 
-      <p>Click here to get a random greeting from around the world!:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div id = "one-page">
+        <div id = "greeting-section" class = "panel">
+            <p id = "greeting-text"> Click here to get a random greeting from around the world! </p>
+            <button id = "greeting-button" onclick="addRandomGreeting()"> Hello </button>
+            <div id="greeting-container"> </div>
+        </div>
+
+        <div id = "education-section" class = "panel">
+            <div id = "school">
+                <ul> <p class = "title"> University of California, Berkeley </p>
+                    <li> Computer Science B.A. </li>
+                    <li> Bioengineering minor </li>
+                    <li> Concentration in Computational Neuroscience</li>
+                    </li>
+                </ul>
+            </div>
+
+            <div id = "coursework">
+                <ul> <p class = "title"> Coursework </p>
+                    <li> Data Structures and Algorithms </li>
+                    <li> Machine Architecture </li>
+                    <li> Operating Systems and Systems Programming </li>
+                    <li> Discrete Mathematics and Probability Theory </li>
+                    <li> Advanced Probability and Random Processes </li>
+                    <li> Signals and Systems </li>
+                    <li> Teaching Techniques for Computer Science </li>
+                </ul>
+            </div>
       
-      <p>Click here to say hello!:</p>
-      <button onclick="fetchMessage()">Fetch Message:</button>
-      <div id="fetch-container"></div>
-
-      <form action="/data" method="POST">
-        <label for="comment">Enter a Comment:</label>
-        <input type="text" id = "comment" name = "comment">
-        <br/><br/>
-        <input type="submit" />
-      </form>
- 
-      <ul id="comment-container"></ul>
+        </div>
+
+        <div id = "activities-section" class = "panel">
+            <div id = "TA" class = "activity">
+                <ul> <p class = "title">TA for EECS 16B </p>
+                    <li> Manage logistics for assignments, exams, discussions, and grading for 500 students. Teach 20-30 students twice a week. </li>
+                </ul>
+            </div>
+            <div id = "research" class = "activity">
```

Comment:
Can you explain why you want to have each item as a full-fledged list inside `<div>` tags? Instead of just four list items?

Reply:
I think this is an artifact of a previous design iteration, I'll remove the outer containers since they don't do anything now.